<launch>
    <node pkg="robot2077_basic" name="robot_avoidance" type="robot_avoidance"/>
    <node pkg="robot2077_basic" name="robot_move" type="robot_move"/>
    <node pkg="robot2077_listen" name="user_listen" type="user_listen"/>
</launch>